html2asketch/helpers/utils.js

```diff
@@ -96,3 +96,15 @@ export const makeImageFill = (url, patternFillType = 1) => {
 
   return result;
 };
+
+export const calculateResizingConstraintValues = (first, ...rest) => rest.reduce((acc, item) => acc & item, first);
```

Comment:
From further investigation it seems sketch will just ignore any values that are either in conflict with one another e.g. fixed left, right and width or undefined and treat it as `none`. 

Is it risky to rely on sketch's internal error handling for this or should we be explicit about it ourselves?

Reply:
> From further investigation

Ah, good to know! I feared it may cause some weird behaviour in Sketch.

> Is it risky to rely on sketch's internal error handling for this or should we be explicit about it ourselves?

IMO it will be much easier for us to debug any issues in the future if we throw for all invalid values here (especially that Sketch fails silently).